### ISSUE TITLE
Night time price (SHF Price2) not working even though times are set

### ISSUE THREAD

**Issue by Sappher** [T3m3z/spotprices2ha#22]
Hi, and firstly, thank you a great sensor package!

I'm quite new to HA but I've gotten used to the basics and with these sensors I've gotten great tools for different automations. However, it seems like the calculation for night time electricity transfer fees isn't working.

I have set up the variables:
![image](https://github.com/T3m3z/spotprices2ha/assets/57065537/ae8d475c-f1a4-48d6-a188-0d0777da34fd)

But it always just uses the Price1 value, even if time is between 22:00 and 07:00:
![image](https://github.com/T3m3z/spotprices2ha/assets/57065537/aed16174-0b0f-4017-a93e-a1de3bd63b01)
![image](https://github.com/T3m3z/spotprices2ha/assets/57065537/a4cecb59-54f9-42b0-9838-a9e9581f7e6a)
![image](https://github.com/T3m3z/spotprices2ha/assets/57065537/b1ea92a8-6617-47e7-a40c-0b7292c30fe6)

I've tried with version 0.11.1 and just updated to 0.2.0 but it still has the same issue.

**Comment by Sappher** [comment]
I took a look at the code for shf_data and it seems like I needed to input the prices other way around: Price1 for night time prices and Price2 for daytime prices. This is because it compares the values as hours and of course in my setup the day changes (between times 22 and 07), making the compare not really valid. I coudln't figure out a clean way to make the compare take this in account either so I just changed the mirrorred the values and it works fine now because day time transfer fee is within a day period (7-22).

So I think this can be closed, maybe update the documentation or if you do find a way to make a reliable compare code - even better!
